docs/source/notebooks/Image_Tut.ipynb

```diff
@@ -0,0 +1,468 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Image Classification with Cleanlab, PyTorch & Skorch"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial will use Cleanlab to find potential label errors in the MNIST handwritten digit image dataset. This dataset contains 60,000 images of handwritten digits from 0 to 9. We'll find *more than a thousand* highly suspicious examples with label errors and edge cases by the end of this tutorial. \n",
+    "\n",
+    "**Overview of what we'll do in this tutorial:**\n",
+    "\n",
+    "- Build a simple PyTorch neural net and wrap it with Skorch to make it scikit-learn compatible.\n",
+    "\n",
+    "- Compute the cross-validated predicted probabilities, ``pyx``, sometimes known as the out-of-sample predicted probabilities.\n",
+    "\n",
+    "- Generate a list of potential label errors with Cleanlab's ``get_noise_indices``.\n",
+    "\n",
+    "- Build and train a more robust model than the vanilla model after adapting it with Cleanlab's ``LearningWithNoisyLabels``. \n",
+    "\n",
+    "**Data:** https://www.openml.org/d/554"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Install the required dependencies"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "``%%capture`` is a magic function to hides the cell's output."
```

Comment:
To save space, can this instead be a comment on the same line as the `%%capture` call (not sure this is doable)?

Reply:
`%%capture` can't have comments after it in the same line and it has to be the first command in the cell. I can remove the description entirely since it's trivial?

Reply:
can just leave as is